server/app.ts

```diff
@@ -55,7 +54,7 @@ export default function createApp(services: Services): express.Application {
   app.use('/change-establishment', establishmentRoutes(services))
   app.use('/prisoner', prisonerRoutes(services))
   app.use('/search', searchRoutes(services))
-  app.use('/altSearch', altSearchRoutes(services))
+  app.use('/search', searchRoutes(services))
```

Comment:
You've got the same route in twice.

Reply:
I don't understand what's gone on here, locally I've deleted this line

edit: reloading VS code now showed both lines, not sure what went on there (so has been removed in most recent commit)